Personal Token follow up: Update transactions.md (#473)
opencollective/opencollective-rest: docs/transactions.md

> Create an Api Key and pass it in the URL parameters as `apiKey`.

Reviewer: ```suggestion
Create a Personal Token and pass it in the URL parameters as `personalToken`.
```

Reviewer: We also need to make sure to support that in the code itself.

> You can create new Personal token from the following page: `https://opencollective.com/{userSlug}/admin/for-developers`

Reviewer: I would explain how to get there too. Like: "Go to your personal settings, then go to the "For Developers" section."
